### ISSUE TITLE
[AnalyticalTable]: Accessibility Related: Scroll bar should not be focused with Tab key

### ISSUE THREAD

**Issue by knandan15** [SAP/ui5-webcomponents-react#7062]
### Is your feature request related to a problem?

As per suggested guidelines by Accessibility Team, the scroll bar in the Analytical Table should be a non-interactive element and should not get focused upon pressing of Tab key.

Right now, when Tab key is pressed, the scroll bar gets highlighted and later navigation up/down the table can be done with arrow-keys.

<img width="1361" alt="Image" src="https://github.com/user-attachments/assets/5b7e0b07-9af1-4b5a-a80c-bbbac88d7216" />

### Describe the solution you'd like

As per suggested guidelines by Accessibility Team, the scroll bar in the Analytical Table should be a non-interactive element and should not get focused upon pressing of Tab key. Same happens in SAP Fiori apps.

### Declaration

- [ ] I’m not disclosing any internal or sensitive information.
